tools/github-actions/upload-build-output/readme.md

```diff
@@ -0,0 +1,7 @@
+# Upload build output action
+
+## Overview
+Upload a GitHub artifact containing all `dist` outputs using upload-artifact GitHub action
```

Comment:
could specify it's a single zip file that contains all the dist folders inside the current working dir